SMTP  not work change port 465

Jellyfin version:  10.11.8
Enable ssl/tls: yes
port: 465 (smpts)
Jellyfin-Newsletters-plugin: 1.6.1.0

Hello—when attempting to switch from STARTTLS (587) to SMTPS (465), the email is not sent using the test function, and the following log is generated.

[NLP]: [2026-04-16] :: [19:13:27] - [ERROR] An error has occured: System.IO.IOException: Connection timed out
 ---> System.Net.Sockets.SocketException (110): Connection timed out
   at MailKit.Net.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   --- End of inner exception stack trace ---
   at MailKit.Net.NetworkStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   at MailKit.Net.Smtp.SmtpStream.ReadAhead(CancellationToken cancellationToken)
   at MailKit.Net.Smtp.SmtpStream.ReadResponse(CancellationToken cancellationToken)
   at MailKit.Net.Smtp.SmtpClient.PostConnect(Stream stream, String host, Int32 port, SecureSocketOptions options, Boolean starttls, CancellationToken cancellationToken)
   at MailKit.Net.Smtp.SmtpClient.Connect(String host, Int32 port, SecureSocketOptions options, CancellationToken cancellationToken)
   at Jellyfin.Plugin.Newsletters.Clients.Email.SmtpMailer.SendTestMail(String configurationId)


Despite having the TLS option enabled, it does not appear to be functioning correctly and consistently "looks for" STARTTLS. Is there a way to force it to use SMTPS properly?

--- a/Jellyfin.Plugin.Newsletters/Clients/Email/smtp.cs
+++ b/Jellyfin.Plugin.Newsletters/Clients/Email/smtp.cs
@@ -154,7 +154,7 @@ public class SmtpMailer(IServerApplicationHost appHost,
             using (var client = new SmtpClient())
             {
                 client.CheckCertificateRevocation = false;
-                var secureOptions = enableSSL ? SecureSocketOptions.StartTls : SecureSocketOptions.None;
+                var secureOptions = enableSSL ? SecureSocketOptions.Auto : SecureSocketOptions.None;
                 client.Connect(smtpAddress, portNumber, secureOptions);
                 if (emailConfig.UseAuthentication)
                 {
@@ -362,7 +362,7 @@ public class SmtpMailer(IServerApplicationHost appHost,
                     {
                         client.Timeout = smtpTimeout;
                         client.CheckCertificateRevocation = false;
-                        var secureOptions = enableSSL ? SecureSocketOptions.StartTls : SecureSocketOptions.None;
+                        var secureOptions = enableSSL ? SecureSocketOptions.Auto : SecureSocketOptions.None;
                         client.Connect(smtpAddress, portNumber, secureOptions);
                         if (emailConfig.UseAuthentication)
                         {

--- a/Jellyfin.Plugin.Newsletters/Configuration/configPage.html
+++ b/Jellyfin.Plugin.Newsletters/Configuration/configPage.html
@@ -1154,7 +1154,7 @@
                     html += '          <input id="email-ssl-' + config.Id + '" type="checkbox" is="emby-checkbox" ' + (config.EnableSsl !== false ? 'checked' : '') + ' onchange="updateEmailConfigFromUI(\'' + config.Id + '\')" />';
                     html += '          <span>Enable SSL/TLS</span>';
                     html += '        </label>';
-                    html += '        <div class="fieldDescription checkboxDescription">Secure the connection to the SMTP server. Most modern services require this.</div>';
+                    html += '        <div class="fieldDescription checkboxDescription">Secure the connection to the SMTP server using SSL/TLS or STARTTLS automatically based on the port.</div>';
                     html += '      </div>';
                     html += '      <div class="checkboxContainer checkboxContainer-withDescription" style="flex: 1; min-width: 200px;">';
                     html += '        <label class="emby-checkbox-label">';

